civicrm_resources/civicrm_resources.module

```diff
@@ -59,6 +61,46 @@ function civicrm_resources_add_resources($extention_path, $req_files = []) {
   }
 }
 
+/**
+ * Filters files into css or js
+ *
+ * @param array $files
+ * @return array
+ */
+function filter_files($files) {
```

Comment:
Functions are shared in the global namespace and a generic name like this can easily cause conflicts. Please prefix them with the module name: `civicrm_resources_filter_files`. Also, this seems like a function that is not supposed to be used by other modules. In this case, I recommend you also prefix it with an underscore to indicate it is "private"